src/Swagger/Serializer/DocumentationNormalizer.php

```diff
@@ -488,9 +488,13 @@ private function getType(string $type, bool $isCollection, string $className = n
      */
     private function computeDoc(Documentation $documentation, \ArrayObject $definitions, \ArrayObject $paths) : array
     {
+        /** @var Symfony\Component\Routing\RequestContext $requestContext */
+        $requestContext = $this->urlGenerator->getContext();
```

Comment:
In my view, it'd be undesirable to introduce yet another interface. Let's make use of `Psr\Http\Message\UriInterface` from PSR-7 :smile: